### PR TITLE
Actually empty the open list

### PR DESCRIPTION
**Pull request by worktycho**

### CODE REVIEW

File: src/Mobs/Path.cpp [hunk 1]
```diff
@@ -245,7 +245,7 @@ void cPath::FinishCalculation()
 	}
 
 	m_Map.clear();
-	m_OpenList.empty();
+	m_OpenList = std::priority_queue<cPathCell *, std::vector<cPathCell *>, compareHeuristics>{};
```
Comment (SafwatHalaby) [inline src/Mobs/Path.cpp]: Isn't it supposed to be `( )`?

Comment (worktycho) [inline src/Mobs/Path.cpp]: its a new c++11 syntax. It avoids issues with most vexing parse.

Comment (SafwatHalaby) [inline src/Mobs/Path.cpp]: Oh, thanks.

**Comment by SafwatHalaby** [comment]
I'd say it's pretty trivial yet important, and should be merged right away.

**Pull request merged**